src/chronometer.js

```diff
@@ -1,33 +1,61 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
+    this.minutes = 0;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;  
+        if(typeof printTimeCallback === 'function'){
+          printTimeCallback(this.currentTime);
+        }; 
+    }, 1000);
   }
 
   getMinutes() {
-    // ... your code goes here
-  }
+   if(this.currentTime === 0){
+    return 0;
+   } 
+      if(this.currentTime > 0){
+        this.minutes = Math.floor(this.currentTime / 60);
+      }
+    return this.minutes;
+  } 
 
   getSeconds() {
-    // ... your code goes here
+   if(this.currentTime === 0){
+      return 0;
+   }  if(this.currentTime > 0){
+        return this.currentTime % 60;
+   }
+    return this.currentTime;
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+   if(value < 10) {
+      return '0' + value;
+    } else{
+        return value.toString();
+    }
   }
 
   stop() {
-    // ... your code goes here
+    clearInterval(this.intervalId);
   }
 
   reset() {
-    // ... your code goes here
+    return this.currentTime = 0;
   }
 
   split() {
-    // ... your code goes here
+    const min = this.getMinutes();
+    const sec = this.getSeconds();
+    const formattedMin = computeTwoDigitNumber(minutes);
```

Comment:
You forgot the `this` keyword here and also used the wrong variable name, instead os minutes should be `min`